adagrams/game.py

```diff
@@ -1,11 +1,89 @@
+import random
+# module will let us randomly choose characters in specified sample range
+
 def draw_letters():
-    pass
+    """
+    Build a hand of 10 letters for the user. Returns an array of strings, one letter long.
+    The num of strings included in the array 
+    cannot be more than the num alloted in the letter table
+    """
+    LETTER_POOL = {
+        'A': 9, 'B': 2, 'C': 2, 'D': 4, 'E': 12, 'F': 2, 'G': 3, 'H': 2, 'I': 9, 'J': 1,
+        'K': 1, 'L': 4, 'M': 2, 'N': 6, 'O': 8, 'P': 2, 'Q': 1, 'R': 6, 'S': 4, 'T': 6, 'U': 4,
+        'V': 2, 'W': 2, 'X': 1, 'Y': 2, 'Z': 1
+    }
+    letter_to_draw = list(LETTER_POOL.keys())
```

Comment:
```suggestion
    letter_to_draw = list(LETTER_POOL)
```

We can omit `keys()` as the default iterator for dictionaries are its own keys. This means the `list` function will iterate through `LETTER_POOL` and create a list of its keys.